### ISSUE TITLE
The intuition of performing the same strong transformation to the same image twice?

### ISSUE THREAD

**Issue by pm25** [LeeHyuck/ABCtorch#2]
Hi, thanks for this great work!

In this code it performs the same strong transformation to the same image twice. And in the training step it also computes the loss of these two images. I'm aware that the two images are not identically the same (output of strong transformation will be slightly different in each time). I didn't find this mention in the paper, therefore I'm wondering are there any intuitions of why doing this? Thanks.

### performs the same strong transformation to the same image twice:
```python
class TransformTwice:
    def __init__(self, transform, transform2):
        self.transform = transform
        self.transform2 = transform2

    def __call__(self, inp):
        out1 = self.transform(inp)
        out2 = self.transform2(inp)
        out3 = self.transform2(inp)
        return out1, out2, out3
```

**Comment by LeeHyuck** [comment]
Sorry for being late

the strong transformation makes stochastic transformation

so the two strongly augmented images are different
